features/306.ngsild.BigDataSpark.feature

```diff
@@ -0,0 +1,132 @@
+        And  I set header NGSILD-Tenant to openiot
+        And  I set the body request as described in <file>
+        And  I perform the request
+        Then  I receive a HTTP "201" response code
+        Examples:
+            | description                | file            |
+            | notify of animal locations | 01.request.json |
+
+    # Request 2 -
+  Scenario: Check the subscriptions for Spark Logger to ngsi-ld
+    When  I send GET HTTP request to "http://localhost:1026/ngsi-ld/v1/subscriptions/"
+    And   I set header NGSILD-Tenant to openiot
+    Then  I receive a HTTP "200" response code
+
+    # Populate things....
+     # Somehow I need to add data to Sensors so they can add data to Cratedb
+    # This is not in tutorial, but it is the way to add data without Web dashboard
+    ## Example post: 12:53:47 PM HTTP POST http://iot-agent:7896/iot/d?i=filling001&k=854782081 f|0.45
+    Scenario Outline: Communicating with IoT Devices: Using Actuators
+    When  I prepare a POST HTTP request to "http://localhost:7896/iot/d?k=<key_value>&i=<sensor>"
+    And   I set header fiware-service to openiot
+    And   I set header fiware-servicepath to /
+    And   I set header Content-Type to text/plain
+    And   I set simple sensor values as described in "<sensor_value>"
+    And   I perform the request
+    Then  I receive a HTTP "200" response code
+    And   I wait "1" seconds
+    Examples:
+        | sensor_value                                                | key_value   | sensor     |
+        | d\|SOWING\|gps\|13.387,52.56\|s\|8                          | 1067386313  | tractor001 |
+        | d\|AT_REST\|bpm\|61\|gps\|13.357,52.515\|s\|0               | 110990      | pig001     |
+        | d\|AT_REST\|bpm\|50\|gps\|13.411,52.468\|s\|0               | 98699       | cow001     |
+        | d\|MOVING\|gps\|13.387,52.561\|s\|7                         | 1067386313  | tractor001 |
+        | d\|AT_REST\|bpm\|60\|gps\|13.359,52.516\|s\|0               | 110990      | pig001     |
+        | d\|GRAZING\|bpm\|53\|gps\|13.41,52.467\|s\|0                | 98699       | cow001     |
+        | d\|IDLE\|gps\|13.3698,52.5163\|s\|0                         | 1067386313  | tractor001 |
+        | d\|WALLOWING\|bpm\|66\|gps\|13.359,52.514\|s\|5             | 110990      | pig001     |
+        | d\|GRAZING\|bpm\|53\|gps\|13.41,52.467\|s\|0                | 98699       | cow001     |
+        | d\|AT_REST\|bpm\|66\|gps\|13.3986,52.5547\|s\|5             | 110990      | pig001     |
+        | d\|AT_REST\|bpm\|53\|gps\|13.3987,52.5547\|s\|0             | 98699       | cow001     |
+
+  # After sending data for Tractor and Devices, we wait one minute until we could have some
+  # output.
+  #
+  Scenario: Logger - checking the output
+    Given I wait "60" seconds
+    When  I Compare next lines in terminal spark-run-jar at least I can find 1 in stdout with a timeout 10 matching filename 01.expected_on_terminal.txt
```

Comment:
When  I Compare next lines in terminal spark-run-jar and at least I can find "1" in stdout with a timeout of "10" seconds matching filename "01.expected_on_terminal.txt"

It is not clear for me, are you waiting the text "01.expected_on_terminal.txt" or matching the content of the file "01.expected_on_terminal.txt"?

Reply:
See the comment in the PR #32 about redefinition of this Scenario.

Reply:
Defined as a TechDebt in the issue #34